submissions/pozzitive11/popup/index.html

```diff
@@ -0,0 +1,136 @@
+                  <img class="menu__item-img" src="./images/videos.png" alt="videos">
+                  <p class="menu__item-descr">Videos</p>
+                </a>
+                <a href="#" class="menu__item">
+                  <img class="menu__item-img" src="./images/search.png" alt="search">
+                  <p class="menu__item-descr">Search</p>
+                </a>
+                <a href="#" class="menu__item">
+                  <img class="menu__item-img" src="./images/photos.png" alt="photo">
+                  <p class="menu__item-descr">Photo</p>
+                </a>
+                <a href="#" class="menu__item">
+                  <img class="menu__item-img" src="./images/apps.png" alt="apps">
+                  <p class="menu__item-descr">Apps</p>
+                </a>
+                <a href="#" class="menu__item">
+                  <img class="menu__item-img" src="./images/documents.png" alt="document">
+                  <p class="menu__item-descr">Document</p>
+                </a>
+                <input id="menu-extra" class="menu__extra-input" type="checkbox">
+                <label class="menu__extra-btn" for="menu-extra">More</label>
+                <div class="menu__extra">
+                  <a href="#" class="menu__item">
+                    <img class="menu__item-img" src="./images/chat.png" alt="chat">
+                    <p class="menu__item-descr">Chat</p>
+                  </a>
+                  <a href="#" class="menu__item">
+                    <img class="menu__item-img" src="./images/classroom.png" alt="classroom">
+                    <p class="menu__item-descr">Classroom</p>
+                  </a>
+                  <a href="#" class="menu__item">
+                    <img class="menu__item-img" src="./images/collection.png" alt="collection">
+                    <p class="menu__item-descr">Collection</p>
+                  </a>
+                  <a href="#" class="menu__item">
+                    <img class="menu__item-img" src="./images/mars.png" alt="mars">
+                    <p class="menu__item-descr">Mars</p>
+                  </a>
+                  <a href="#" class="menu__item">
+                    <img class="menu__item-img" src="./images/blogs.png" alt="blogs">
+                    <p class="menu__item-descr">Blogs</p>
+                  </a>
+                  <a href="#" class="menu__item">
+                    <img class="menu__item-img" src="./images/promo.png" alt="promos">
+                    <p class="menu__item-descr">Promos</p>
+                  </a>
+                  <a href="https://github.com/kottans/frontend" class="menu__extra-link">
+                    Even more from Kottans
+                  </a>
+                </div>
+              </div>
+            </div>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__btn">
+              <img class="avatar" src="images/bell.png" alt="Notification icon">
```

Comment:
For decorative images, like icons, not necessarily populate `alt` attribute? better left it empty